src/jdk.jlink/share/classes/jdk/tools/jlink/internal/PerfectHashBuilder.java

```diff
@@ -246,7 +246,7 @@ public void generate() {
                 if (bucket.getSize() != 1) {
                     // Attempt to pack entries until no collisions occur.
                     if (!collidedEntries(bucket, count)) {
-                        // Failed to pack. Meed to grow table.
+                        // Failed to pack. Need to grow table.
```

Comment:
Change copyright date to mark this change.

Reply:
Fixed.